plugin/FormPost.php

```diff
@@ -136,6 +136,19 @@ public function initialCheck(DataFormConfiguration $pConfig, int $formNo)
 		}
 	}
 
+	/**
+	 * @param  DataFormConfiguration  $pFormConfiguration
+	 *
+	 * @return bool|mixed|void
+	 */
+	protected function getRecipient( DataFormConfiguration $pFormConfiguration ) {
```

Comment:
Why is the method attached to this class? It only uses the `DataFormConfiguration`, right?

Reply:
Right, what do you mean? If you don't want to write there, I think I can write this function in `DataFormConfiguration` class instead of in `FormPost`.